Controllers/controller.py

```diff
@@ -106,3 +111,20 @@ def open_file_dialog(self):
             url = file_dialog.selectedUrls()[0]
             self._media_player.setSource(url)
             self._media_player.play()
+
+    @Slot()
+    def position_changed(self):
+        """Command the video player to change state of the position"""
```

Comment:
This comment seems inaccurate. The function is changing the value of the progress bar slider, not the video player. Here is perhaps a better comment:
"""
Sets the value of the progress bar slider based on the position of the media player.
"""